Sources/Requests/TextDocumentSourceKitOptionsHandler.swift

```diff
@@ -278,61 +260,13 @@ final class TextDocumentSourceKitOptionsHandler {
                 continue
             }
 
-            // Skip index store path arguments (handled later)
-            if arg.contains("-index-store-path") {
-                if index + 1 < count, rawArguments[index + 1].contains("indexstore") {
-                    index += 2
-                    continue
-                }
-            }
-
-            // Skip const-gather-protocols arguments
-            if arg.contains("-Xfrontend"), index + 1 < count {
-                let nextArg = rawArguments[index + 1]
-                if nextArg.contains("-const-gather-protocols-file")
-                    || nextArg.contains("const_protocols_to_gather.json")
-                {
-                    index += 2
-                    continue
-                }
-            }
```

Comment:
@josh-arnold-1 Did you run into issues with this one? iirc Xcode adds it last time I checked. I remember running into errors pointing to this file but the root cause was actually something else. But let me know if I'm wrong